afisha-server/server/config/config.js

```diff
@@ -5,7 +5,11 @@ const config = {
     test: 'testing',
     prod: 'production',
     port: process.env.PORT || 8800,
-    dataBaseUrl: process.env.DATABASEURL || 'mongodb://localhost/eventAPI'
+    dataBaseUrl: process.env.DATABASEURL || 'mongodb://localhost/eventAPI',
+    expireTime: 30, //1 day
+    secrets: {
+        jwt: process.env.JWT || 'hooters'
```

Comment:
Never ever use default values with environment variables. It's very hard to debug if something goes wrong.